src/pages/Signup.js

```diff
@@ -11,17 +11,44 @@ import Typography from "@mui/material/Typography"
 import Container from "@mui/material/Container"
 import { createTheme, ThemeProvider } from "@mui/material/styles"
 import { Link as RouterLink } from "react-router-dom"
+import firebase from "../Firebase/FirebaseConfig"
 
 const theme = createTheme()
 
 export default function SignUp() {
+  const [signup, setSignup] = React.useState({
```

Comment:
Out of curiosity was there a specific reason you switched from using `new FormData(event.currentTarget)` in `handleSubmit` to get the form values to using `useState`?

Reply:
I originally copied the Signup component from the material ui templates :)
When I went back to link the `<Signup/>` component  to Firebase, I saw `new FormData(event.currentTarget)` but was unfamiliar with it. So far I have only used `useState` with forms so I decided to change it to useState instead. What it not needed for me to do that if `new FormData(event.currentTarget)` was already there ?